core/reactor.h

```diff
@@ -876,6 +877,17 @@ void _lf_notify_workers(void);
  */
 bool _lf_is_blocked_by_executing_reaction(void);
 
+/**
+ * Check the deadline of the currently executing reaction against the
+ * current physical time. If the deadline has passed, invoke the deadline
+ * handler and return true. Otherwise, return false.
+ * 
+ * @param self The self struct of the reactor.
+ * @return true if the specified deadline has passed.
```

Comment:
I think usually there is only one @return statement, which would read like this:
```
@return True if the specified deadline has passed and false otherwise.